internal/driver/webui_test.go

```diff
@@ -119,7 +119,10 @@ func TestWebInterface(t *testing.T) {
 		for count := 0; count < 2; count++ {
 			wg.Add(1)
 			go func() {
-				http.Get(path)
+				res, err := http.Get(path)
```

Comment:
Why not fail the test if the call fails? Do you expect it to ever fail?

Reply:
Done.